addons/mass_mailing_sms/models/mailing_list.py

```diff
@@ -27,6 +27,21 @@ def action_view_contacts_sms(self):
         action['context'] = dict(action.get('context', {}), search_default_filter_valid_sms_recipient=1)
         return action
 
+    def action_send_new_sms(self):
```

Comment:
If we can re-use 'action_send_mailing' in mail, maybe this one could be named 'action_send_mailing_sms' to match naming ?